rust/gotham/Cargo.toml

```diff
@@ -4,8 +4,8 @@ version = "0.1.0"
 authors = ["Isaac Whitfield <iw@whitfin.io>"]
 
 [dependencies]
-gotham = "~0.5"
-gotham_derive = "~0.5"
+gotham = "~0.6"
+gotham_derive = "~0.6"
```

Comment:
is it needed to define explicitly deps other than `gotham` ?

Reply:
Yes, serde is required if you use path or query parameters. `serde_derive` could be removed because it is re-exported by serde.

Reply:
I mean `gotham_derive`. I'm not aware of `best-practices` in **rust** 😛

Reply:
gotham does not currently re-export `gotham_derive`, but we might do that in the future

Reply:
regarding best practises though - it looks like you are still using rust 2015 edition, if you want I can make a PR and update to 2018 edition

Reply:
it will be awesome 🎉